zio-http/src/test/scala/zhttp/internal/HttpGen.scala

```diff
@@ -9,6 +10,8 @@ import zio.stream.ZStream
 import zio.test.{Gen, Sized}
 
 object HttpGen {
+  private def toDom(s: String): Dom = Dom.element(s)
```

Comment:
Why wrap a function into a method?

Reply:
It will be converted to a function during eta-expansion.